Makefile

```diff
@@ -94,15 +94,19 @@ generate-test-keys:
 		frapsoft/openssl
 
 .PHONY: user-agent-start
-user-agent-start: clean user-agent-wasm-docker credential-mediator-docker generate-test-config generate-test-keys
+user-agent-start: clean user-agent-wasm-docker credential-mediator-docker did-method-cli generate-test-keys
```

Comment:
We don't need `generate-test-config` anymore?
if Yes, then why that target still exists in code base?

Reply:
Should have removed it, fixed.